tests/bson/bug0313.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+PHPC-313: BSON should throw when encountering 64-bit integer on 32-bit platform
+--SKIPIF--
+<?php if (4 !== PHP_INT_SIZE) { die('skip Only for 32-bit platform'); } ?>
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/tools.php';
+
+$tests = [
+    '{ "x": { "$numberLong": "4294967294" }}',
+    '{ "x": { "$numberLong": "4294967295" }}',
+    '{ "x": { "$numberLong": "9223372036854775807" }}',
+    '{ "longFieldName": { "$numberLong": "4294967294" }}',
+    '{ "longFieldName": { "$numberLong": "4294967295" }}',
+    '{ "longFieldName": { "$numberLong": "9223372036854775807" }}',
+];
+
+foreach ($tests as $json) {
+	throws(function() use ($json) { var_dump(toPHP(fromJSON($json))); }, 'MongoDB\Driver\Exception\InvalidArgumentException');
```

Comment:
The integer macros in `phongo_compat.h` is evidently my fault from https://github.com/mongodb/mongo-php-driver/commit/6bcba59cb57a92f091b29e4f11e1c99a96eaed95. I'll clean that up after this and #435 are merged if any offending lines remain.

That's not relevant to the PHPT indentation, though.